.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,48 @@
 Thanks for opening a Pull Request (PR), here's a few guidelines as to what we need in your PR before we review it.
```

Comment:
Per our discussion:
```suggestion
<!-- 
Thanks for opening a Pull Request (PR), here's a few guidelines as to what we need in your PR before we review it.
-->
```

Reply:
I tried making all the changes, as I understood them, in https://github.com/mozilla/addons-server/pull/21711  (Doesn't need merging, can be closed, it was just an easy way to edit the whole file)

Reply:
(Maybe I should have made the changes in suggestions like you did so the discussion was in one place 😬 )

Reply:
Your changes are large enough that I think it's a good idea to have them as a separate PR, even though it can be a bit confusing

Reply:
Left a summary comment on your PR @eviljeff let's align on what to do with it and then come back here to finish things off.